src/tests/SignUpForm.test.js

```diff
@@ -0,0 +1,123 @@
+import '@testing-library/jest-dom/extend-expect';
+import { render } from "@testing-library/react";
+import { screen, waitFor, within } from '@testing-library/dom';
+import {BrowserRouter as Router} from 'react-router-dom';
+import user from '@testing-library/user-event'
+import UserRegistration from '../components/SignUpForm';
+
+describe('FormCreateChecking tests', () => {
+    const onSubmit = jest.fn();
+    const nextBtn = jest.fn();
+    const previousBtn = jest.fn();
+
+    beforeEach(() =>{
+        onSubmit.mockClear();
+        nextBtn.mockClear();
+        previousBtn.mockClear();
+        render(
+            <Router>
+                <UserRegistration/>
+            </Router>
+        );
+    });
+
+    it('Should render the User Registration form', () => {
+        const view = render(
+            <Router>
+                <UserRegistration/>
+            </Router>
+        );
+        expect(view).toBeDefined();
+    });
+
+    it('Should submit the form when all fields are valid...', () => {
```

Comment:
The three dots aren't necessary. What I meant in my comment was that you could add something else after the suggested message.